qaframework-bdd-tests/cypress/support/commands.js

```diff
@@ -76,7 +76,7 @@ Cypress.Commands.add('generateLabResults', (uuid) => {
         location: '44c3efb0-2583-4c80-a79e-1f756a03c0a1',
         encounterProviders: [
           {
-            provider: "b917e518-9186-4981-90a0-f95625c18641",
+            provider: "43fa3b4c-6dc6-4d8d-87c4-e6715399ae38",
```

Comment:
Does the uuud changes simultaneously

Reply:
there were no providers in our instance, hence we get null [here](https://github.com/openmrs/openmrs-module-webservices.rest/blob/master/omod-1.9/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/resource/openmrs1_9/EncounterResource1_9.java#L78) hence mysql blocks saving this encounter.  

I actually had to create a provider manually from a person at dev3.openmrs.org and referenced its uuid here

Reply:
Sounds good.